pom.xml

```diff
@@ -162,30 +155,9 @@
                     <artifactId>maven-compiler-plugin</artifactId>
                     <version>${versions.maven-compiler-plugin}</version>
                     <configuration>
-                        <source>8</source>
-                        <target>8</target>
-                    </configuration>
-                </plugin>
-                <plugin>
-                    <groupId>org.codehaus.mojo</groupId>
-                    <artifactId>animal-sniffer-maven-plugin</artifactId>
-                    <version>1.23</version>
-                    <configuration>
-                        <signature>
-                            <groupId>org.codehaus.mojo.signature</groupId>
-                            <artifactId>java18</artifactId>
-                            <version>1.0</version>
-                        </signature>
+                        <source>11</source>
+                        <target>11</target>
```

Comment:
I wouldn't be too quick to abandon the Java 8 users. We can try [modulemaker](https://github.com/raphw/modulemaker-maven-plugin) maven plugin to generate the module-info, so that we don't need to upgrade our source to 11.